src/main/java/org/folio/service/loader/BlockingRecordLoaderService.java

```diff
@@ -1,51 +1,56 @@
 package org.folio.service.loader;
 
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.clients.impl.SourceRecordStorageClient;
+import org.folio.util.OkapiConnectionParams;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collection;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
-import org.folio.clients.SynchronousOkapiClient;
-import org.springframework.stereotype.Service;
-
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-
 /**
  * Synchronous implementation of #RecordLoaderService that uses blocking http client.
  */
 @Service
 public class BlockingRecordLoaderService implements RecordLoaderService {
 
-  private final SynchronousOkapiClient client;
-
-  public BlockingRecordLoaderService(SynchronousOkapiClient client) {
-    this.client = client;
-  }
+  @Autowired
+  private SourceRecordStorageClient client;
 
   @Override
-  public SrsLoadResult loadMarcRecords(List<String> uuids) {
-    Optional<JsonObject> optionalRecords = client.getByIds(uuids);
+  public SrsLoadResult loadMarcRecords(List<String> uuids, OkapiConnectionParams okapiConnectionParams) {
+    Optional<JsonObject> optionalRecords = client.getByIds(uuids, okapiConnectionParams);
     SrsLoadResult srsLoadResult = new SrsLoadResult();
     optionalRecords.ifPresent(records -> populateLoadResult(uuids, records, srsLoadResult));
```

Comment:
if there are no SRS records for any of the  UUIDs, the srsLoadResult will be entirely empty. The instanceswithoutsrs will not be populated. That should be done here